src/main/java/uk/gov/hmcts/probate/config/ApplicationConfiguration.java

```diff
@@ -47,7 +53,7 @@ public ObjectMapper objectMapper(ObjectMapper objectMapper) {
         objectMapper.registerModule(module);
```

Comment:
instead of modifying the existing objectmapper, could you create a config for feign like this:
https://github.com/hmcts/probate-orchestrator-service/blob/master/src/main/java/uk/gov/hmcts/probate/client/SubmitServiceConfiguration.java
You can create you new object mapper with registered modules in there.